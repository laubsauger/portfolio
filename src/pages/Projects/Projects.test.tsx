import * as React from 'react';
import Projects from './Projects';
import {render} from "@testing-library/react";

describe('Projects', () => {
  it('renders', () => {
    const { asFragment } = render(<Projects />);
    expect(asFragment()).toMatchSnapshot();
  });
});