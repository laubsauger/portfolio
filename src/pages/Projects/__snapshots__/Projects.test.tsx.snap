// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders 1`] = `
<DocumentFragment>
  <div>
    Projects
  </div>
</DocumentFragment>
`;
